pig/src/org/partiql/pig/domain/Utils.kt

```diff
@@ -0,0 +1,67 @@
+package org.partiql.pig.domain
```

Comment:
I was thinking that they should probably be private and live in the only place they are used. If needed multiple places we pull them out.

Reply:
They're in the `domain` package because that's where they are in the tests ;) (this code is almost verbatim copy-pasted from [`test/org/partiql/pig/domain/Util.kt`](https://github.com/partiql/partiql-ir-generator/blob/main/pig/test/org/partiql/pig/domain/Util.kt)).  I don't really have a strong preference here though - if you want, I'll move it.

Reply:
Although these functions are used by more than just the new `Ion` target though: they are also used by the parser tests.

Reply:
I also notice that you've *only* moved the limited set of functions that you need for this this to the `src` dir but left the other related functions in the `test` dir.  Since they are so closely related, I would generally prefer to move them as a single unit.

Reply:
Other things you could consider:

- Making these regular instance functions on their respective types (instead of extension functions)
- ... or, since there's a number of them, they aren't just "utils" anymore, put them into a file with a more specific name.

Reply:
Ah, okay, I see what these are for now.  The reason I copied them (instead of deduplicating them) is to avoid having the tests depend on implementation details (since they are asserting against a specific format).  I'll just dedup them.